code/go/0chain.net/smartcontract/zcnsc/factory_helpers_test.go

```diff
@@ -44,7 +44,7 @@ func (n *Authorizer) Verify(sig, hash string) (bool, error) {
 }
 
 func CreateDefaultTransactionToZcnsc() *transaction.Transaction {
-	return CreateAddAuthorizerTransaction(defaultClient, MakeMockStateContext(), tokens)
+	return CreateAddAuthorizerTransaction(defaultClient, MakeMockStateContext(), tokens) //nolint: typecheck
```

Comment:
I assume the problem is still the ambiguity between ` state.StateContextI,` and ` *mockStateContext`. I guess we need to either explicitly declare the second parameter or change MakeMockStateContext's return type.

Reply:
We will ignore all lint errors on test files, so should be fine.